File: pclassic/stubs-2016/ENIACAccumulating.py
def eniac_accumulating(a):
    """

    :param a: list of length n + 1, which represents an n digit 0-padded positive integer
    :return: a, incremented by 1
    """
    #write ur code here

    return a


def parse_file_and_call_function():
    with open("ENIACAccumulatingIN.txt", "r") as f:
        for line in f:
            a = [int(digit) for digit in line.split(',')]
            print (','.join([str(digit) for digit in eniac_accumulating(a)]))


if __name__ == "__main__":
    parse_file_and_call_function()

File: pclassic/stubs-2016/ENIACAccumulating.java
import java.io.FileNotFoundException;
import java.io.FileReader;
import java.util.Scanner;

public class ENIACAccumulating {
    public static void main(String[] args) throws FileNotFoundException {
     // Before submitting, make sure the main method hasn't been changed!
        Scanner sc = new Scanner(new FileReader("ENIACAccumulatingIN.txt"));
        while(sc.hasNextLine()) {
            String line = sc.nextLine();
            String[] elements = line.split(",");
            int[] a = new int[elements.length];
            for(int i = 0; i < elements.length; i++) {
                a[i] = Integer.parseInt(elements[i]);
            }
            int[] output = accumulate(a);
            String s = Integer.toString(output[0]);
            for(int j = 1; j < output.length; j++) { 
                s += "," + output[j];
            } 
            System.out.println(s);
        }
        sc.close();
    }
    
    public static int[] accumulate(int[] a) {
        //write ur code here
        return a;
    }
}
